app/about-us/mobile.tsx

```diff
@@ -1,33 +1,57 @@
+'use client';
+
 import AboutImage from '@/images/about-image.png';
-import LeftSplash from '@/images/about-us-left-splash.png';
 import VideoBG from '@/images/about-us-video-bg.png';
 import VideoSplash from '@/images/about-us-video-splash.png';
+import { motion } from 'framer-motion';
 import Image from 'next/image';
 
+const textContent = [
+  {
+    title: 'TED',
+    desc: `TED merupakan sebuah organisasi nirlaba dengan semangat “Ideas Worth Spreading” yang berfokus pada Technology, Entertainment, dan Design. TED pertama kali dibentuk pada tahun 1984 dan menjadi ruang bagi individu-individu dengan semangat yang sama untuk menyebarkan ide-ide brilian di seluruh dunia.`,
+  },
+  {
+    title: 'TEDx',
+    desc: `TEDx merupakan events yang mengolaborasikan komunitas lokal untuk berbagi ide dengan konsep yang ada pada TED sehingga memicu diskusi dan koneksi yang mendalam. TEDx dengan label ikonik bersimbol “x” bermakna bahwa events tersebut diselenggarakan secara independen dengan lisensi dari TED.`,
+  },
+  {
+    title: 'TEDxUniversitasBrawijaya',
+    desc: `TEDxUniversitasBrawijaya merupakan salah satu dari TEDx event yang diselenggarakan secara independen dengan lisensi TEDx yang diinisiasi komunitas lingkup Universitas Brawijaya.`,
+  },
+];
+
+const textVariant = {
```

Comment:
also this separate to another files... also learn how to create object in typescript.. example

Reply:
import { Variants } from 'framer-motion';

const textVariant: Variants = {
  visible: { filter: `blur(0rem)` },
  hidden: { filter: `blur(0.2rem)` },
};